cropper/src/main/java/com/canhub/cropper/CropImageOptions.kt

```diff
@@ -38,522 +38,312 @@ open class CropImageOptions : Parcelable {
 
     /** The initial scale type of the image in the crop image view  */
     @JvmField
-    var scaleType: CropImageView.ScaleType
+    val scaleType: CropImageView.ScaleType = CropImageView.ScaleType.FIT_CENTER,
 
     /**
      * if to show crop overlay UI what contains the crop window UI surrounded by background over the
      * cropping image.<br></br>
      * default: true, may disable for animation or frame transition.
      */
     @JvmField
-    var showCropOverlay: Boolean
+    val showCropOverlay: Boolean = true,
 
     /**
      * If enabled, show a text label on top of crop overlay UI, which gets moved along with the cropper
      */
     @JvmField
-    var showCropLabel: Boolean
+    val showCropLabel: Boolean = false,
 
     /**
      * if to show progress bar when image async loading/cropping is in progress.<br></br>
      * default: true, disable to provide custom progress bar UI.
      */
     @JvmField
-    var showProgressBar: Boolean
+    val showProgressBar: Boolean = true,
 
     /** The color of the progress bar. Only works on API level 21 and upwards. */
     @JvmField
     @ColorInt
-    var progressBarColor: Int
+    val progressBarColor: Int = COLOR_PURPLE,
 
     /**
      * if auto-zoom functionality is enabled.<br></br>
      * default: true.
      */
     @JvmField
-    var autoZoomEnabled: Boolean
+    val autoZoomEnabled: Boolean = true,
 
     /** if multi-touch should be enabled on the crop box default: false  */
     @JvmField
-    var multiTouchEnabled: Boolean
+    val multiTouchEnabled: Boolean = false,
 
     /** if the crop window can be moved by dragging the center; default: true  */
     @JvmField
-    var centerMoveEnabled: Boolean
+    val centerMoveEnabled: Boolean = true,
 
     /** The max zoom allowed during cropping.  */
     @JvmField
-    var maxZoom: Int
+    val maxZoom: Int = 4,
 
     /**
      * The initial crop window padding from image borders in percentage of the cropping image
      * dimensions.
      */
     @JvmField
-    var initialCropWindowPaddingRatio: Float
+    val initialCropWindowPaddingRatio: Float = 0.1f,
 
     /** whether the width to height aspect ratio should be maintained or free to change.  */
     @JvmField
-    var fixAspectRatio: Boolean
+    val fixAspectRatio: Boolean = false,
 
     /** the X value of the aspect ratio.  */
     @JvmField
-    var aspectRatioX: Int
+    val aspectRatioX: Int = 1,
 
     /** the Y value of the aspect ratio.  */
     @JvmField
-    var aspectRatioY: Int
+    val aspectRatioY: Int = 1,
 
     /** the thickness of the guidelines lines in pixels. (in pixels)  */
     @JvmField
-    var borderLineThickness: Float
+    val borderLineThickness: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 3f, Resources.getSystem().displayMetrics),
 
     /** the color of the guidelines lines  */
     @JvmField
     @ColorInt
-    var borderLineColor: Int
+    val borderLineColor: Int = Color.argb(170, 255, 255, 255),
```

Comment:
Let's please extract this value so we know what colour we expect here. just for reading